client/blocks/plugins-scheduled-updates/index.tsx

```diff
@@ -98,16 +105,30 @@ export const PluginsScheduledUpdates = ( props: Props ) => {
 						'Streamline your workflow with scheduled updates, timed to suit your needs.'
 					) }
 				>
-					{ context === 'list' && ! hideCreateButton && onCreateNewSchedule && (
-						<Button
-							__next40pxDefaultSize
-							icon={ plus }
-							variant={ canCreateSchedules && siteHasEligiblePlugins ? 'primary' : 'secondary' }
-							onClick={ onCreateNewSchedule }
-							disabled={ ! canCreateSchedules || ! siteHasEligiblePlugins }
-						>
-							{ translate( 'Add new schedule' ) }
-						</Button>
+					{ context === 'list' && ! hideButtons && (
```

Comment:
If there are two schedules, the Notification management button won't show up in this case.

Reply:
Nice catch. I'll fix it.